### ISSUE TITLE
Checks failed on tasks 4 and 5

### ISSUE THREAD

**Issue by M1-Elmasry** [Davenchy/alx-files_manager#37]
unfortunately there is a 2 failed checks in task 4 and 5 in task 5.

**Comment by Davenchy** [comment]
I will investigate after pushing task 7 #32, as probably it is a formatting problem.

**Comment by M1-Elmasry** [comment]
i don't think that it's wrong output checks, anyway, i checked the task 4 endpoints (`/connect`, `/disconnect`, `/users/me`) response and status codes like required

**Comment by M1-Elmasry** [comment]
it's wrong output checks, anyway, i checked the task 4 endpoints (/connect, /disconnect, /users/me) response and status codes same as required

**Comment by Davenchy** [comment]
Also, task 6 has 3 fails

**Comment by Davenchy** [comment]
Fails on **task 8**: 5, 8, 12

**Comment by Davenchy** [comment]
**Latest changes**

- **Task 4**: 7, 11
- **Task 5**: 7, 9, 10, 11, 13
- **Task 6**: 5, 11, 13
- **Task 8**: 5, 6, 7, 8, 9, 11, 12, 14, 15, 16 - 3/10 pass/fail

**Comment by Davenchy** [comment]
After reverting code in #43 and #44 .
**Task 8 fails**: 6, 7, 11, 14, 15, 16.
7 passes / 6 fails

**Comment by Davenchy** [comment]
Task 6 have been resolved

**Comment by Davenchy** [comment]
All old fails (7, 11) have been resolved, but check 5 fails.
I downloaded all the test cases, and they all pass. So, I have no idea what is the problem!?

**Comment by M1-Elmasry** [comment]
When i was writing tests in task 10 i noticed there is test cases passes in the first try and then fails, after some debugging i knew that's happened because in the first try redis and db are clear there is no records in them, so try to dump some data into redis and db and run tests again

**Comment by Davenchy** [comment]
> When i was writing tests in task 10 i noticed there is test cases passes in the first try and then fails, after some debugging i knew that's happened because in the first try redis and db are clear there is no records in them, so try to dump some data into redis and db and run tests again

This solves many problems while testing on my machine, but about the checker, I checked the testing codes for each check, and the tests do drop and recreate the db and collections and fill them with initial data. So, I don't think this affect the checker, but it does affect our local environments.
